frontend/architecture.md

```diff
@@ -1,9 +1,18 @@
 # architecture for frontend
 
-> This will be the top level architecture for the frontend of the project-cms repo.
+> This will be the top level architecture for the frontend of the galericms repo.
 
 ## TODO:
-- [ ] Create arch
+- [ ] Create technical arch
```

Comment:
Prevent using unnecessary abbreviations like arch. Just type it out to architecture 😛